tests/integration-tests/tests/ad_integration/test_ad_integration/test_ad_integration/ad_stack.yaml

```diff
@@ -391,11 +249,7 @@ Outputs:
             - VpcId
   PublicSubnetIds:
     Value:
-      Fn::Join:
-        - ""
-        - - Ref: PublicSubnetOne
-          - ","
-          - Ref: PublicSubnetTwo
+      Ref: PublicSubnetOne
```

Comment:
Is it guaranteed that PublicSubnetOne and PrivateSubnetOne will be from the same AZ?

Reply:
It is guaranteed by vpc_stack